app/javascript/shared/components/EmojiPicker.res

```diff
@@ -0,0 +1,93 @@
+type emojiEvent = {
+  id: string,
+  native: string,
+  unifield: string,
+  shortcodes: string,
+}
+
+type args = {
+  title: string,
+  ref: React.ref<Js.Nullable.t<Dom.element>>,
+  theme: string,
+  onEmojiSelect: emojiEvent => unit,
+  data: Js.Json.t,
+}
+
+type t
+@module("emoji-mart") @new external picker: args => t = "Picker"
```

Comment:
I'm not sure what the purpose of the _opaque type_ `t` is. Instead of returning the _opaque type_, I think it might be cleaner for this `picker` function's type signature to be `args => unit`.

```suggestion
@module("emoji-mart") @new external picker: args => unit = "Picker"
```

That way, there's no need for the `let _ = picker({...})`. That can simply be `picker({...})` because ReScript won't bother us about any expression that returns `()` (unit).